lib/Doctrine/ORM/Mapping/Driver/XmlDriver.php

```diff
@@ -255,11 +255,20 @@ public function loadMetadataForClass($className, ClassMetadata $metadata)
 
         if (isset($xmlRoot->embedded)) {
             foreach ($xmlRoot->embedded as $embeddedMapping) {
+                $columnPrefix = isset($embeddedMapping['column-prefix'])
```

Comment:
I'd say you're correct, I'm thinking better now since my last comment lol, makes sense.